### DISCUSSION TITLE
blog/a-closer-look-shared-module/

### DISCUSSION THREAD

**Discussion by giscus** [nartc/tnc-scully#37] [Announcements]
# blog/a-closer-look-shared-module/

Personal blog by Chau Tran

https://nartc.me/blog/a-closer-look-shared-module/

**Comment by hakimio** [comment]
Links in your RSS feed are broken: nartc.me/feed.xml . The articles are linked to "nartc.meblog".

**Reply by nartc** [reply]
Thanks for the headsup. This is probably due to Astro v2 removing trailing slash. I'll update it asap
